keyboards/planck/keymaps/mguterl/readme.md

```diff
@@ -0,0 +1,19 @@
+# Michael Guterl's Planck Layout
+
+I have a very similar [keymap](/keyboards/preonic/keymaps/mguterl) for my
+Preonic.
+
+Check out my [userspace](/users/mguterl) for the custom keycodes that are used
+in keymap.c.
+
+At a very high level I use 4 layers:
+
+* Qwerty - Letters
+* Util   - Vimkeys for navigation and other convenient things
+* Raise  - Numbers and F keys
+* Lower  - Symbols
+
+```sh
+make planck/rev5:mguterl:dfu         # For Planck rev5 or earlier and Planck Light
+make planck/rev6:mguterl:dfu-util    # For Planck rev6
```

Comment:
Actually, we have `:flash` now.  This handles the detection and will attempt to flash using the "correct" method based on the controller and bootloader set.

Reply:
This is awesome, thanks for the tip!